.github/workflows/test.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       - run: yarn lint
       - run: yarn audit --groups dependencies
       - run: yarn test:unit --coverage
+      - run: yarn test:e2e
+      - run: yarn test:all
```

Comment:
We don't need to call this command here as part of CI because both sets of tests are already called individually in the previous steps, so this would be redundant.
```suggestion
```